Librarian/Series.js

```diff
@@ -36,21 +33,20 @@ Series.prototype.fetch = (done)=> {
   return;
 };
 
-
 /**
 SeriesコレクションからBookListコレクションを取得する
 **/
-Series.prototype.join = function(seriesItems, done){
-  var conditions = {
+Series.prototype.join = function join(seriesItems, done) {
+  const conditions = {
     _id: {
-      $in: seriesItems.map(function(seriesItem){
+      $in: seriesItems.map((seriesItem)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).